packages/thumbprint-react/components/ButtonRow/__snapshots__/test.tsx.snap

```diff
@@ -3,8 +3,6 @@
 exports[`adds \`dataTest\` prop 1`] = `
 <ButtonRow
   dataTest="Duck"
-  isStackedBelowSmall={false}
```

Comment:
Any idea why these get removed from the snapshot?

Reply:
Not sure. I need to look into that. I would expect them to still be there because they're now in the default function arguments